platform/example.anchor-config.yaml

```
@@ -0,0 +1,100 @@
+stellar:
+  anchor:
+    # Configure the application from this file after it is loaded in memory
+    # If `config` is in-memory, the yaml file contains all settings for the server.
+    config: in-memory
+    #
+    # Configures how the applications read the
+    #
+    app-config:
+      type: config-spring-property                # Activate [config-spring-property] module
+      settings: app-config                        # The location of the configuration data
+    data-access:
+      type: data-spring-jdbc                      # Activate [config-spring-jdbc] module.
+      settings: data-spring-jdbc-settings         # The location of the configuration data in this file.
+    logging:
+      type: logging-logback
+      settings: logging-logback-settings
+
+#
+# Application settings
+#
+app-config:
+  # general
+  app:
+    stellarNetworkPassPhrase: Test SDF Network ; September 2015
+    hostUrl: http://localhost:8080
+    languages: en
+    horizonURI: https://horizon-testnet.stellar.org
+    assets: assets-test.json
+    jwtSecretKey: secret
+
+  # sep-1
+  sep1:
+    enabled: true
+    stellarFile: sep1/stellar-wks.toml
+
+  # sep-10
+  sep10:
+    enabled: true
+    homeDomain: localhost:8080
+    clientAttributionRequired: false
+    clientAttributionAllowList: lobstr.co,preview.lobstr.co
+    authTimeout: 900
+    jwtTimeout: 86400
+    signingSeed: SAX3AH622R2XT6DXWWSRIDCMMUCCMATBZ5U6XKJWDO7M2EJUBFC3AW5X
+
+  # sep-24
+  sep24:
+    interactiveJwtExpiration: 300
+    interactiveUrl: http://localhost:8080/ref/sep24/interactive
+
+  payment-gateway:
+    #
+    # Payment Circle configurations
+    #
+    circle:
+      name: "circle_payment"
+      enabled: true
+      circleUrl: http://payment.circle.com
+      secretKey: This-is-a-secret-key
+      horizonUrl: http://horizon.stellar.org
```

Can this inherit from the app-config? Not sure we could have variables in a yaml file.

Also, we should default to testnet:
```suggestion
      horizonUrl: https://horizon-testnet.stellar.org
```

---

changed.